static/src/js/components/GranuleResultsHighlights/__tests__/GranuleResultsHighlights.test.js

```diff
@@ -77,5 +77,21 @@ describe('GranuleResultsHighlights component', () => {
         expect(enzymeWrapper.find('.granule-results-highlights__item-title').text()).toEqual('producer_granule_id_1')
       })
     })
+
+    describe('when granules have no start time nor end time', () => {
+      test('display not provided instead of date', () => {
+        const { enzymeWrapper } = setup({
+          isLoading: false,
+          isLoaded: true,
+          granules: [{
+            // formattedTemporal is intentionally absent from the granule object
+            // in order to mimic the behavior of the app when it is run
+            title: 'producer_granule_id_1'
+          }]
+        })
+        expect(enzymeWrapper.find('.granule-results-highlights__temporal-row').at(0).text()).toEqual('StartNot Provided')
```

Comment:
This might be a little easier to understand if you broke it up to `.toEqual('Start')` and `.toEqual('Not Provided')` and treat the end time the same way.

Reply:
done.